mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/KitPvp.java

```diff
@@ -22,66 +22,196 @@
 import gg.solarmc.loader.Transaction;
 import gg.solarmc.loader.data.DataObject;
 import gg.solarmc.loader.impl.SQLTransaction;
+import gg.solarmc.loader.schema.tables.records.KitpvpKitsOwnershipRecord;
 import gg.solarmc.loader.schema.tables.records.KitpvpStatisticsRecord;
 import org.jooq.DSLContext;
-import org.jooq.impl.DSL;
+import org.jooq.Result;
 
-import java.math.BigDecimal;
-import java.util.Objects;
+import java.util.HashSet;
 import java.util.Set;
 
 import static gg.solarmc.loader.schema.tables.KitpvpStatistics.*;
+import static gg.solarmc.loader.schema.tables.KitpvpKitsOwnership.*;
 
 public class KitPvp implements DataObject {
 
-    private volatile BigDecimal kills;
-    private volatile BigDecimal deaths;
-    private volatile BigDecimal assists;
+    private volatile Integer kills;
```

Comment:
why Integer rather than int? Kills, deaths, and assists are non-null in the schema

Reply:
Fixed, i'll make sure to read the schema next time